tests/server/routes.test.js

```diff
@@ -0,0 +1,33 @@
+const index = require('../../server/db/index')
+const server = require('../../server/server')
+const request = require('supertest')
+
+test('/', (done) => {
+  request(server)
+    .get('/')
+    .expect(200)
+    .end((err, res) => {
+      expect(err).toBeNull()
+      done()
+    })
+})
+
+test('/:id', (done) => {
+  request(server)
+    .get('/:id')
+    .expect(200)
+    .end((err, res) => {
+      expect(err).toBeNull()
+      done()
+    })
+})
+
+test('/api/v1/auth', (done) => {
```

Comment:
Why is a test for `/api/v1/auth` in here? It's great that you're testing it but I don't see it listed in the original ticket #38. If you want to add some additional tests unrelated to the ticket, it would be best to create a new ticket for this.

Reply:
Please remove this test and I'll approve the changes.